app/views/admin/configs/show.html.erb

```diff
@@ -161,6 +161,59 @@
                   How can people create an account on your community?
                 </p>
 
+                <section>
+                  <div class="form-group">
```

Comment:
In order to implement the design team's [mockup](https://user-images.githubusercontent.com/816402/96671965-f5909180-1317-11eb-990b-9bf4cabfa5cd.jpg) for Enabling/Disabling Authentication Providers, I employed the use of a hidden text field.

Whenever the auth_providers' `Enable`, `Undo` or `Disable` buttons are clicked, handlers in `config_controller.js` fire off such that this hidden text field populates with a string of auth_providers that should remain enabled.

This string is passed as `params[:auth_providers_to_be_enabled]` to the `configs_controller.rb`, where it is processed appropriately, before updating the `SiteConfig.authentication_providers` value.

It is possible for ALL `auth_providers` to be disabled (this was NOT the case when we had them as a dropdown list). However, I have built 2 important guards in:
- If there is just one `auth_provider` enabled, it cannot be disabled unless Email Authentication is on.
- In the case when more than 1 `auth_provider` is currently enabled, if Email Auth is off and the user disables all `auth_providers` at once, this disabling IS NOT processed in the controller (because it would effectively lock everyone out of the forem).

Finally, when `invite_only_mode` is enabled, this action disables the